wordpress-mobile/WordPress-iOS
# Feature/15178 today widget unconfigured view

Fixes #15178

Adds the unconfigured view for Today Widgets. Here are two example screenshots.

 light appearance | dark appearance
-- | --
<img width="380" src="https://user-images.githubusercontent.com/34376330/101396526-10e83980-3891-11eb-97e9-24e45bebf66b.png"> | <img width="380" src="https://user-images.githubusercontent.com/34376330/101396559-1f365580-3891-11eb-8695-3bbde97e6864.png">



To test:
Prerequisite: if you had previously tested PRs from the iOS 14 widgets project, better to start from scratch (delete the app).

1. Install the app
2. Before logging in, install the widget and verify that the unconfigured view is displayed
3. Log in and verify that the default site stats for that account are shown in the widget
4. also verify that long press on the widget and then "Edit widget" allows you to configure the widget
5. Logout and verify that the unconfigured view appears again in the widget
6. Login with another account and verify that the default site stats for that account appear and that you can do the configuration as you did in step n.4.

**Known issues:**

when switching between accounts and trying to configure widgets, before changing the configuration you might see a site in the preview that belongs to the previously logged in account. This is an issue related to `WidgetKit` that we are trying to figure out. It seems related to the TODO at line 68 of WordPressHomeWidgetToday.swift and we'll open a separate issue for that

PR submission checklist:
